build.fsx

```diff
@@ -69,6 +69,8 @@ Target "NuGet" (fun _ ->
         | "ApiCheck" ->
             CopyFile libDir (buildDir @@ "ApiCheck.dll")
             CopyFile libDir (buildDir @@ "ApiCheck.XML")
+            CopyFile libDir (buildDir @@ "YamlDotNet.dll")
+            CopyFile libDir (buildDir @@ "YamlDotNet.XML")
```

Comment:
This will work only as long as the consuming project does not include YamlDotNet itself. That might lead to version conflicts. A better approach is to add the dependency as nuget dependency.

For this you can edit the packages variable in this file (`build.fsx`)